tests/helper_util.py

```diff
@@ -53,6 +52,8 @@ def do_pickle(obj, func=(lambda x : x), copyable=True, reprable=True):
                         obj = eval(repr(obj)).  This is often a desirable property of objects,
                         but it may not always be possible (or efficient). [default: True]
     """
+    import coord
```

Comment:
```suggestion
    # This import needs to be hidden from the top-level of this helper module so
    # that JAX-GalSim can test its implementation of the functionality in coord
    # using this test suite. The jax-galsim test suite, instead of vendoring the
    # tests from coord and galsim, uses the upstream test suites. It replaces all
    # imports of galsim and coord with imports of jax_galsim. This helper file
    # gets reimported after the initial replacement, and the reimport of coord
    # breaks the replacement. By delaying the import, we avoid this breakage.
    # - MRB 2024-07-26
    import coord
```